docs/manual/advanced-features.tex

```diff
@@ -285,6 +285,23 @@
 Section~\ref{climb-to-top}.
 
 
+\subsectionAndLabel{Default qualifier on constructor declarations}{constructor-defaults}
```

Comment:
There is already text in the introduction about annotations on constructors (https://checkerframework.org/manual/#annotations-on-constructor-invocations).   I think it would be better to consolidate the information there rather than adding a new section (especially since there are no cross-references between the two sections).

Note that other pull requests have also changed that part of the manual.